.gitignore

```diff
@@ -22,3 +22,4 @@ google.json
 
 # CLI completions
 completions/
+.DS_Store
```

Comment:
Can you also put this in a separate chore commit and separate it from the "CLI completions" sections?